atd-vzd/migrations/default/1705620819250_crashes_add_near_trauma_center_fields/up.sql

```diff
@@ -0,0 +1,4 @@
+-- Add new trauma center fields from CRIS to the crashes table
+ALTER TABLE public.atd_txdot_crashes
+ADD COLUMN near_trauma_center_id integer NULL,
+ADD COLUMN near_trauma_center_distance double precision NULL;
```

Comment:
i looked at all the recent records and their `record_json` fields in the `atd_txdot_changes` table to make the assumption that `near_trauma_center_id` will always be an integer and `near_trauma_center_distance` seems to always be a number with 3 digits. I thought [double precision](https://www.postgresql.org/docs/current/datatype-numeric.html) was a safe type choice bc we use it for a lot of other floating point fields.